test/__snapshots__/step-function.localstack.test.ts.snap

```diff
@@ -0,0 +1,386 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[` 1`] = `
```

Comment:
Is this missing a description?

Reply:
The self-mutation should remove these. I was comparing the snapshots outside of the tests and jest didn't like that (even though I was giving it a name).